airbyte-integrations/bases/base-typing-deduping-test/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/BaseTypingDedupingTest.java

```diff
@@ -617,31 +694,36 @@ private static List<AirbyteMessage> readMessages(final String filename, final St
    * make edits here, you probably want to also edit there.
    */
 
-  // These contain some state, so they are instanced per test (i.e. cannot be static)
-  private Path jobRoot;
-  private ProcessFactory processFactory;
```

Comment:
these didn't need to actually be fields, and in fact need to be isolated per-process to avoid clobbering the destination connector inputs (i.e. catalog/config). I moved these declarations into the methods, so each process now gets its  own jobRoot + processFactory.

(possibly the processfactory can be shared? but I didn't bother testing that, and I don't think it really matters)